api/src/main/java/com/kodemon/api/dto/UserDTO.java

```diff
@@ -19,6 +19,10 @@
     private String lastName;
     @JsonProperty(required = true)
     private Date dateOfBirth;
+    @JsonProperty(required = true)
+    private boolean isAdmin;
+    @JsonProperty(required = true)
+    private boolean isBlocked;
```

Comment:
Same as the above, also with default value set to false. We do not want to allow other users to set themselves (or others) as blocked. (IMHO)